src/style.css

```diff
@@ -1 +1,18 @@
 /* styles go here */
+body {
+  font-family: "Times New Roman", Times, serif;
+  font-size: 16px;
```

Comment:
Do we need this? 
I see 10 point in checklist: https://github.com/mate-academy/layout_html-form/blob/master/checklist.md and as I understand we can't to set this